microsoft-edge/webview2/concepts/frames.md

```diff
@@ -0,0 +1,344 @@
+
+
+<!-- ====================================================================== -->
+## Starting work with frames
+<!--
+## Starting work with frames by subscribing to the FrameCreated event to obtain a frame
+## Subscribing to the FrameCreated event to obtain a frame -->
+
+The first step to interacting with frames in your app is to subscribe to the `FrameCreated` event to obtain a frame object.
+The `FrameCreated` event is fired whenever a new frame is created.
+Once you have the frame object, you can then use this object to monitor changes or interact with this specific frame.
+
+It's important to monitor the lifetime of a frame by subscribing to the `CoreWebView2Frame.Destroyed` event, because when the frame is destroyed, your app can no longer reference that frame.  Frames get created and destroyed during each new webpage navigation.  Use the `CoreWebView2Frame.IsDestroyed` method to check whether the frame still exists.
+
+To receive the frame object, you must subscribe to the `FrameCreated` event.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2.FrameCreated Event](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2.framecreated)
+* [CoreWebView2FrameCreatedEventArgs Class](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2framecreatedeventargs)
+
+* [CoreWebView2Frame.Destroyed Event](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2frame.destroyed)
+   * [CoreWebView2Frame.IsDestroyed Method](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2frame.isdestroyed)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2.FrameCreated Event](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#framecreated)
+   * [CoreWebView2FrameCreatedEventArgs Class](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2framecreatedeventargs)
+
+* [CoreWebView2Frame.Destroyed Event](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2frame?view=webview2-winrt-1.0.1370.28#destroyed)
+   * [CoreWebView2Frame.IsDestroyed Method](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2frame?view=webview2-winrt-1.0.1370.28#isdestroyed)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2_4::FrameCreated event (add](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2_4#add_framecreated), [remove)](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2_4#remove_framecreated)
+   * [ICoreWebView2FrameCreatedEventArgs interface](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2framecreatedeventargs)
+
+* [ICoreWebView2Frame::Destroyed event]<!-- todo - API Ref exists?  mentioned in next topic: -->
```

Comment:
@vbryh-msft - for Win32 is there an API Ref page/url for ICoreWebView2Frame::Destroyed event like for .NET?  That event is mentioned, but not linked, from the below linked page.
https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2frame?view=webview2-1.0.1370.28#isdestroyed

.net:
https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2frame.destroyed

Reply:
Outdated comment thread.  Latest changes:
https://github.com/MicrosoftDocs/edge-developer/pull/2251/files

Plan: Ask Devs if there's a gap in API Ref docs that were gen'd.